pwndbg/arch.py

```diff
@@ -18,6 +18,7 @@
 import pwndbg.typeinfo
 
 current = 'i386'
+mode	= {CS_MODE_ARM:"ARM",CS_MODE_THUMB:"THUMB",None:""}[None]
```

Comment:
It would be better to have this dict only once in the code, so maybe it would be good to do:
```python
MODES = {CS_MODE_ARM: "ARM", CS_MODE_THUMB: "THUMB", None: ""}
```

and then:
```python
mode = MODES[None]
```